app/coffee/infrastructure/ExpressLocals.coffee

```diff
@@ -61,11 +61,18 @@ module.exports = (app, webRouter, apiRouter)->
 
 	webRouter.use (req, res, next)-> 
 
+		cdnBlocked = req.query.nocdn == 'true' or req.session.cdnBlocked
+
+		if cdnBlocked and !req.session.cdnBlocked?
+			ip = req.ip || req.socket?.socket?.remoteAddress || req.socket?.remoteAddress
```

Comment:
Hmm, that's a lie, I think it's from old express. Anyway, to keep our code up to date and clean, I think it should just be `req.ip`.